src/main/java/gregtech/common/covers/CoverFluidVoiding.java

```diff
@@ -63,9 +66,17 @@ public ModularUI createUI(EntityPlayer player) {
 
         this.fluidFilter.initUI(20, primaryGroup::addWidget);
 
-        ModularUI.Builder builder = ModularUI.builder(GuiTextures.BACKGROUND, 176, 100 + 82)
+        primaryGroup.addWidget(new CycleButtonWidget(10, 92, 80, 18, this::isWorkingEnabled, this::setWorkingEnabled,
+                "cover.voiding.label.disabled", "cover.voiding.label.enabled")
+                .setTooltipHoverString("cover.voiding.tooltip"));
+
+        primaryGroup.addWidget(new CycleButtonWidget(10, 112, 116, 18,
+                ManualImportExportMode.class, this::getManualImportExportMode, this::setManualImportExportMode)
+                .setTooltipHoverString("cover.universal.manual_import_export.mode.description"));
+
+        ModularUI.Builder builder = ModularUI.builder(GuiTextures.BACKGROUND, 176, 100 + 82 + 16 + 24)
                 .widget(primaryGroup)
-                .bindPlayerInventory(player.inventory, GuiTextures.SLOT, 7, 100);
+                .bindPlayerInventory(player.inventory, GuiTextures.SLOT, 7, 100 + 16 + 23);
```

Comment:
Are you sure about last magical number 23? Wont be magical number 24 better?

Reply:
fixed in https://github.com/GregTechCEu/GregTech/pull/1414/commits/af3efa58c7d31b574e60955c1b036ae4abefede7.